content/en/contribute/code/workflow.md

```diff
@@ -22,6 +22,19 @@ Aim for self-documenting code. Where code cannot be made self-documenting add co
 
 A good workflow would be to work locally, pushing to a remote branch as you make progress, possibly open a draft PR for some initial collaboration on tricky parts, and once everything is done, convert the draft PR to a regular PR to be reviewed.
 
+### Updating The Issue With What You Actually Did
```

Comment:
Do you think before code review is the right place to add the comment? I'd be tempted to do it before AT because the notes/screenshots/etc may change during review, and presumably the reviewer is technical enough not to need it.

Reply:
This would also be a good time to remind the dev to add the "Breaking" and "UI/UX" labels as appropriate.